main.tex

```diff
@@ -215,368 +222,296 @@ \subsection*{Functional variation}
-We label a new cluster ``L1'' representing an outbreak driven by the \texttt{I1527T} allele in combination with one or the other \texttt{V402L} allele.
+A possible explanation for this pattern of divergence is that a gene conversion event has occurred within the gene, bringing a short segment from a different genetic background onto the original genetic background on which the \texttt{L995S} resistance mutation occurred.
 %
-We also label a cluster ``L2'' capturing a set of haplotypes from Kenya carrying the \texttt{M490I} variant, although the fact that these haplotypes all share a recent common ancestor may be a reflection of the unusual demography of the Kenyan population which has experienced a severe population crash and not be due to recent selection for insecticide resistance \cite{Ag1000gConsortium2017}.
+For the remainder of this paper we combine S4 and S5 into a single cluster labelled S4/5.
 %
-As in earlier analyses (Figure \ref{fig:networks}; \cite{Ag1000gConsortium2017}), clusters F1, F4, F5, S2 and S3 all include haplotypes sampled from multiple geographical locations, and F1 includes haplotypes from both species, \textit{An. gambiae} and \textit{An coluzzii}.
+All haplotypes in S4/5 were sampled from Cameroon, and thus considering this as a single genetic background does not imply any new gene flow events.
 %
-Clusters F2, F3, S1, S4/5, L1 and L2 include only haplotypes from a single sampling location.
-%%
-
-
-%% Figure - Overall haplotype age distribution.
+We also found that all haplotypes carrying the \texttt{I1527T} allele were grouped together when clustered by IBD length (cluster L1 in Supplementary Figure @@), which was not the case in fixed window analyses of the \textit{Vgsc} gene (data not shown).
 %
-\begin{figure}[!b]
-  \includegraphics[width=1.1\linewidth,center]{artwork/haplotype_age_dist.pdf}
-  \caption{\textbf{Haplotype IBD distribution}. @@TODO caption.}
-  \label{fig:age_hist}
-\end{figure}
-%%
- 
- 
-%% Figure - Haplotype age tree.
+The moving window analysis showed that haplotypes within the L1 cluster were all nearly identical on the downstream flank of the gene, but separated into multiple clusters on the upstream flank.
+%
+The change in genetic distance occurred between the \texttt{I1527T} and \texttt{V402L} codons, and we know that \texttt{I1527T} occurs in combination with two separate nucleotide substitutions causing \texttt{V402L}, thus the data are consistent with one or both of the \texttt{V402L} alleles having been brought together with an ancestral haplotype carrying \texttt{I1527T} via crossover recombination events.
 %
-\begin{figure}[!b]
-  \includegraphics[width=1.1\linewidth,center]{artwork/fig_hap_tree.png}
-  \caption{\textbf{Clustering of haplotypes by age}. @@TODO caption.}
-  \label{fig:tree}
-\end{figure}
 %%
 
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection*{Haplotype selection}
+\subsection*{Positive selection for resistance alleles}
```

Comment:
I've edited the section on selection. Nothing radical here, just a bit of elaboration and additional explanation.